src/components/Payment/Payment.js

```diff
@@ -35,34 +35,49 @@ class Payment extends Component {
   }
 
   async componentDidMount() {
-    const { t } = this.props;
-    try {
-      const response = await getPaymentMethods();
-      const { paymentMethods } = response.responseData;
-      if (paymentMethods) {
-        if (!paymentMethods.length) {
-          this.setState({
-            generalError: t('Payment methods are not defined')
-          });
-        } else {
+    const { t, availablePaymentMethods } = this.props;
+    if (availablePaymentMethods && availablePaymentMethods.length) {
+      this.setState({
+        paymentMethods: availablePaymentMethods
+      });
+      const defaultMethod = availablePaymentMethods.find(
+        method => method.default
+      );
+      if (defaultMethod) {
+        this.setState({ isPaymentFormDisplayed: true });
+        this.choosePaymentMethod(defaultMethod.id, defaultMethod.methodName);
+      }
+      setData('CLEENG_PAYMENT_METHOD_ID', availablePaymentMethods[0].id);
```

Comment:
if there is a default payment method we should save this one in the local storage, not the first one